nss-day-cohort-41/tabloid-fullstack-shocking-secret-keepers
# Sf user

# Description
Referring to ticket #26 (Reactivating user profile)
1. ability to reactive user profile after deactivation (admin only)
2. ability to view list of deactivated users
3. reactivation returns them back to active users list
4. only see user profile and categories if admin in nav bar 

## Type of change
Please delete options that are not relevant.

# Testing Instructions
1. git fetch --all
2. checkout to branch sf-user
3. from userprofile list page, click details button on a user and deactivate that user (try multiple users)
4. then click 'view deactivated user profiles' from list of all user profiles
5. you should see a list of all the user profiles that have been deactivated
6. click 'reactivate' 
7. you should be brought back to the user profile list page and your user profile entry should reappear on this list with previous access permissions
8. try to log in with user that is not admin; should not be able to see nav links for user profile or categories

# Checklist:
